web/cypress/integration/route/create-route-with-upstream.spec.js

```diff
@@ -84,7 +83,9 @@ context('Create Route with Upstream', () => {
     cy.get('[title=Name]').type(data.routeName);
     cy.contains('Search').click();
     cy.contains(data.routeName).siblings().contains('Edit').click();
-    cy.wait(1000);
+
+    // wait for resource to resolve 
+    cy.wait(500);
```

Comment:
How about timeout?

Reply:
Need to wait for loading here, timeout can't.
![image](https://user-images.githubusercontent.com/72343596/104833181-ba2f4000-58d1-11eb-9030-56c45cc81b68.png)